src/sa_web/static/js/routes.js

```diff
@@ -22,6 +25,14 @@ var Shareabouts = Shareabouts || {};
           startPageConfig,
           filteredRoutes;
 
+      // REFACTOR
+      // This object stores all collections, including shareabouts places and landmarks.
+      // Here, it's initialized with a key-value pair for shareabouts data. Landmark collections
+      // are added below (~line 85)
+      this.collection = {
+        "places": new S.PlaceCollection([])
```

Comment:
You are probably aware, but just in case, the plan is to define a dataset ID in the config, and use it as the key for our collection. So instead of hardcoding "places", we would loop over the Shareabouts datasets and create our key-value pairs that way.